src/meetingsession/MeetingSessionConfiguration.ts

```diff
@@ -173,7 +173,11 @@ export default class MeetingSessionConfiguration {
     } else if (typeof input !== 'object') {
       return input;
     } else if (Array.isArray(input)) {
-      return input.map(this.toLowerCasePropertyNames);
+      if (input.length) {
+        return input.map(this.toLowerCasePropertyNames);
+      } else {
+        return input;
+      }
```

Comment:
I'm not entirely sure what you're trying to fix here, but I suspect the correct thing to do is to move `toLowerCasePropertyNames` out as a free function. Even with your change this use of `this` is incorrect.